README.md

```diff
@@ -7,23 +7,35 @@ Grouplogger is a specialized Stackdriver logging client for writing groups of lo
 ([Stackdriver documentation: Viewing related request log entries](https://cloud.google.com/appengine/docs/flexible/go/writing-application-logs#related-app-logs))
 
 ```go
-var r *http.Request
-var ctx *context.Context
+// Stackdriver requires a non-nil http.Request.
```

Comment:
I guess the most important thing to illustrate is that the request must be non-nil. I think the proposed code is somewhat long, given that it's not about `grouplogger` itself.

Additionally, 

I'd propose something shorter, e.g.:

```go
// Grouplogger requires a non-nil http.Request. 
u, _ = url.Parse("https://api.vimeo.com/")
req := &http.Request{URL: u}
```

+ Shorter, so more focus on the `grouplogger` quick-start.

+ Swapping the example URL is a minor thing, but it's nice to avoid linking to a domain that can be bought and turned into something shady.

+ Avoids suggesting that `grouplogger` *requires* `X-Cloud-Trace-Context` to be set for the request; it doesn't.

Reply:
Good points.

IMO it increases the usefulness of the example code to show how `X-Cloud-Trace-Context` is used, but you're right that we shouldn't suggest that it's required, so I added a comment noting that it's optional.